src/components/ArticleActions.vue

```diff
@@ -9,23 +9,23 @@
     &nbsp;&nbsp;
     <button
       class="btn btn-outline-danger btn-sm"
-      v-on:click="deleteArticle(article.slug)">
+      @click="deleteArticle(article.slug)">
```

Comment:
I think this was still part of your old PR

Reply:
yes, forgot to rebase

Reply:
oh no, is just that I've created a new branch from the previous, with this commit, so it's actually right